pkg/install/install_test.go

```diff
@@ -121,8 +90,10 @@ func TestDeployARMTemplate(t *testing.T) {
 			}
 
 			err := i.deployARMTemplate(ctx, resourceGroup, "test", armTemplate, params)
-			if !reflect.DeepEqual(err, tt.wantErr) {
-				t.Error(err)
+
+			if cloudErr, ok := err.(*api.CloudError); ok &&
+				!reflect.DeepEqual(cloudErr.CloudErrorBody.Message, tt.wantErr) {
+				t.Error(cloudErr.CloudErrorBody.Message)
 			}
```

Comment:
```suggestion
			if err != nil && err.Error() != tt.wantErr ||
				err == nil && tt.wantErr != "" {
				t.Error(err)
			}
```

use the standard pattern.